File: src/main/java/cloud/requestengine/RTCollectionTimeout.java
package cloud.requestengine;

import se.sics.kompics.timer.ScheduleTimeout;
import se.sics.kompics.timer.Timeout;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-04-02
 *
 */

public class RTCollectionTimeout extends Timeout {

	public RTCollectionTimeout(ScheduleTimeout request) {
		super(request);
	}

}

File: src/main/java/cloud/requestengine/RTCollectionTimeout.scala
package cloud.requestengine

import se.sics.kompics.timer.{ScheduleTimeout, Timeout}

/**
 * @author Amir Moulavi
 * @date 2011-07-12
 *
 */
class RTCollectionTimeout(request:ScheduleTimeout) extends Timeout(request)